pyrk/inp/sim_info.py

```diff
@@ -110,6 +131,10 @@ def init_ne(self):
         ne = neutronics.Neutronics(iso=self.iso, e=self.e,
                                    n_precursors=self.n_pg,
                                    n_decay=self.n_dg,
+                                   n_reflector=self.n_ref,
+                                   Lambda_ref=self.Lambda_ref,
+                                   ref_rho=self.ref_rho,
+                                   ref_lambda=self.ref_lambda,
```

Comment:
add new tests where necessary to test neutronics class instantiation using these variables. Does everything propagate? Do all old neutronics methods work with multipoint variables defined?